python/ray/utils.py

```diff
@@ -812,3 +814,25 @@ def get_user():
 def get_function_args(callable):
     all_parameters = frozenset(signature(callable).parameters)
     return list(all_parameters)
+
+
+def _zip_module(path, zip_handler):
+    for from_file_name in path.glob("**/*"):
```

Comment:
In a followup we'll need to add some file exclusions and special case big file handling.